compiler/noirc_frontend/src/hir/comptime/value.rs

```diff
@@ -761,6 +755,255 @@ impl<'token, 'interner> Display for TokenPrinter<'token, 'interner> {
     }
 }
 
+fn print_quoted(
+    tokens: &[Token],
+    indent: usize,
+    interner: &NodeInterner,
+    f: &mut std::fmt::Formatter<'_>,
+) -> std::fmt::Result {
```

Comment:
There's getting to be a lot of value formatting code. Can you move it to it's own sub-module in a new file?